### ISSUE TITLE
False-positive `reportPrivateUsage` for `matplotlib.colormaps`

### ISSUE THREAD

**Issue by bersbersbers** [microsoft/pyright#6429]
**Describe the bug**
`from matplotlib import colormaps` reports
> "colormaps" is private and used outside of the module in which it is declared (reportPrivateUsage)

in `strict` mode.

**Code or Screenshots**

```python
from matplotlib import colormaps
```

**VS Code extension or command-line**
pyright 1.1.335 via pyright-python

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.336, which I just published. It will also be included in a future release of pylance.
